### ISSUE TITLE
wx.Locale: Aragonese does not have wx.LanguageInfo

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#9089]
Hi,

Reported on NvDA translations mailing list and subsequently confirmed: NVDA 2018.4 fails to start properly if UI language is set to Aragonese.

### Steps to reproduce:
1. Install NVDA 2018.4 and current alpha.
2. From NVDA settings/General, switch language to "Aragonese" and press Enter, and confirm if asked to restart.

### Actual behavior:
The following traceback is recorded:

CRITICAL - __main__ (11:26:22.013):
core failure
Traceback (most recent call last):
  File "nvda.pyw", line 233, in <module>
  File "core.pyc", line 390, in main
AttributeError: 'NoneType' object has no attribute 'Language'

### Expected behavior:
NVDA starts properly with language set to Aragonese.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2018.4

#### Windows version:
Windows 10 Version 1809

#### Name and version of other software in use when reproducing the issue:
None

#### Other information about your system:
None

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
Yes - alpha.16458.

### Technical information:
Not all languages come with wx.LanguageInfo, and Aragonese is one of those that  isnt' supported by wxPython natively. Thus see if wxLang itself is something and act accordingly.

Thanks.

**Comment by Adriani90** [comment]
we have reports from the translator that it works fine now. So I think this issue can be closed and can be reopened if it is being reported again.

**Comment by ehollig** [comment]
Fixed from #9092. This was not automatically closed because it went into RC for NVDA 2018.4

**Comment by josephsl** [comment]
Hi,

I'm overriding close status, as this fix is still being tested by some more. I'll close this once 2018.4.1 is released.

Thanks.

**Comment by Adriani90** [comment]
@josephsl did you get further test results from users? Are there still issues with Aragonese or other languages?

**Comment by Adriani90** [comment]
I have tested also on Windows 10 and 8.1 with NVDA 2018.4.1. Everything seems to work properly now. Closing this as works for me since it is also part of NVDA 2018.4.1. We can reopen it if users report this issue again. Thanks.
